extensions/vertx-http/deployment/src/main/java/io/quarkus/vertx/http/deployment/devmode/tests/TestsProcessor.java

```diff
@@ -0,0 +1,135 @@
+        //GET tests/results
+
+        return new DevConsoleRouteBuildItem("tests/status", "GET", new Handler<RoutingContext>() {
+            @Override
+            public void handle(RoutingContext event) {
+                jsonResponse(event);
+                TestSupport.RunStatus status = ts.get().getStatus();
+                TestStatus testStatus = new TestStatus();
+                testStatus.setLastRun(status.getLastRun());
+                testStatus.setRunning(status.getRunning());
+                if (status.getLastRun() > 0) {
+                    TestRunResults result = ts.get().getResults();
+                    testStatus.setTestsFailed(result.getTestsFailed());
+                    testStatus.setTestsPassed(result.getTestsPassed());
+                    testStatus.setTestsSkipped(result.getTestsSkipped());
+                    testStatus.setTestsRun(result.getTestsFailed() + result.getTestsPassed());
+                }
+                event.response().end(JsonObject.mapFrom(testStatus).encode());
+            }
+        });
+    }
+
+    @BuildStep(onlyIf = IsDevelopment.class)
+    DevConsoleRouteBuildItem toggleTestRunner(LaunchModeBuildItem launchModeBuildItem) {
+        Optional<TestSupport> ts = TestSupport.instance();
+        if (!ts.isPresent() || launchModeBuildItem.getDevModeType().orElse(null) != DevModeType.LOCAL) {
+            return null;
+        }
+        //GET tests/status
+        //DISABLED, RUNNING (run id), RUN (run id, start time, nextRunQueued)
+        //GET tests/results
+
+        return new DevConsoleRouteBuildItem("tests/toggle", "POST", new Handler<RoutingContext>() {
+            @Override
+            public void handle(RoutingContext event) {
+                if (ts.get().isStarted()) {
+                    ts.get().stop();
+                } else {
+                    ts.get().start();
+                }
+            }
+        });
+    }
+
+    @BuildStep(onlyIf = IsDevelopment.class)
+    DevConsoleRouteBuildItem runAllTests(LaunchModeBuildItem launchModeBuildItem) {
+        Optional<TestSupport> ts = TestSupport.instance();
+        if (!ts.isPresent() || launchModeBuildItem.getDevModeType().orElse(null) != DevModeType.LOCAL) {
+            return null;
+        }
+        //GET tests/status
+        //DISABLED, RUNNING (run id), RUN (run id, start time, nextRunQueued)
+        //GET tests/results
+
+        return new DevConsoleRouteBuildItem("tests/runall", "POST", new Handler<RoutingContext>() {
+            @Override
+            public void handle(RoutingContext event) {
+                ts.get().runAllTests();
+            }
+        });
+    }
+
+    @BuildStep(onlyIf = IsDevelopment.class)
+    DevConsoleRouteBuildItem handleTestResult(LaunchModeBuildItem launchModeBuildItem) {
+        Optional<TestSupport> ts = TestSupport.instance();
+        if (!ts.isPresent() || launchModeBuildItem.getDevModeType().orElse(null) != DevModeType.LOCAL) {
+            return null;
+        }
```

Comment:
Looks like this `if` is present in 5 methods in this class, should it be extracted to something like `isDevConsoleEnabled(LaunchModeBuildItem)` or something?